app/uk/gov/hmrc/apidocumentation/views/apispecification/resource_content.scala.html

```diff
@@ -119,26 +119,35 @@ <h4 class="govuk-heading-s">Request</h4>
 
 @fields(RequestFields2(meth), "Request table", viewModel.isFieldOptionalityKnown)
 
-@for(resp <- Responses.success(meth)) {
-    <div id="sample-response_@contentID" class="section">
-        @if(resp.headers.nonEmpty) {
-            @headers(resp.headers, viewModel.version, "Response headers", "", ("none", None))
-        }
+@defining(Responses.success(meth)) { successes =>
+    @defining(Responses.error(meth).filter(Responses.hasResponseExample)) { errors =>
+            <div id="sample-response_@contentID" class="section">
 
-        <h4 class="govuk-heading-s">Response</h4>
-        <p class="govuk-body">HTTP status: <code class="code--slim">@HttpStatus(resp.code)</code></p>
-        @for(body <- resp.body) {
-            @for(example <- BodyExamples(body)) {
-                <p class="govuk-body">
-                    @if(example.description.isDefined){@example.description}
-                    @if(example.documentation.isDefined){<div class="markdown-text-styles">@Markdown(example.documentation.get)</div>}
-                    <pre class="code--block">@example.value</pre>
-                </p>
-            }
-        }
-    </div>
+                <h4 class="govuk-heading-s">Responses</h4>
+
+                @for(resp <- (successes ++ errors)) {
+                    <h5 class="govuk-heading-xs">HTTP status @HttpStatus(resp.code)</h5>
```

Comment:
Could we just have 'govuk-heading-s'? The class 'govuk-heading-xs' doesn't exist. Or you could have 'govuk-body govuk-!-font-weight-bold'.